src/index.ts

```diff
@@ -118,7 +118,8 @@ const getAuthTokens = (): IAuthTokens | undefined => {
   try {
     // parse stored tokens JSON
     return JSON.parse(rawTokens)
-  } catch (error) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
probably better to have a type guard here to check which kind of error it is and act bc of the type, instead of disabling the rule (some for the other catch below).

Reply:
There is only one type of error in this case, a syntaxerror, but your right. Problem im having is the linter doesnt allow type annotations other than any or unknown for the catch clause variable

Reply:
Maybe you can try something like 
```
if(error instanceof Error){
    error.message = `Failed to parse auth tokens: ${rawTokens}`
    throw error
}
```
just a suggestion tho

Reply:
Ill check it out, thanks! Do you have any idea why the linter keeps failing? I ran prettier but git isn't detecting any change to commit

Reply:
Maybe take a look at https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#use-unknown-catch-variables

Reply:
I'm having an issue with the catch in the refreshToken function as it seems that a 422 error is not the normal axios error type, I have removed the other 2 occurrences though. Any idea about prettier?